### ISSUE TITLE
gbXML Viewer display

### ISSUE THREAD

**Issue by michaldengusiak** [ladybug-tools/spider-2020#11]
I did some test and had a go with new version...there is some display issue with some
![image](https://user-images.githubusercontent.com/26113670/84136091-b9345700-aa42-11ea-9d55-dc65ccdab63a.png)
![image](https://user-images.githubusercontent.com/26113670/84136245-eed94000-aa42-11ea-8feb-db2dab0f4d74.png)

[SAM_export.zip](https://github.com/ladybug-tools/spider-2020/files/4751507/SAM_export.zip)

**Comment by theo-armour** [comment]
@michaldengusiak 

Yes, I have seen this error as well. I believe the reason is this: Three.js wants the vertices of surface boundaries and hole boundaries to go in opposite directions. The errors may be occurring when all the vertices go in the same direction. I will add a check for this. I have hesitated because checks like this can really slow things down. While I am programming, I may need to load a model dozens of times and waiting even a few seconds can be frustrating. And I hear you. A fix wil arrive one day. ;-)

![image](https://user-images.githubusercontent.com/547626/84347807-5046ff00-ab68-11ea-91ac-e89fb082f16f.png)
